offlineimap/imapserver.py

```diff
@@ -198,7 +198,7 @@ def __md5handler(self, response):
         challenge = response.strip()
         self.ui.debug('imap', '__md5handler: got challenge %s'% challenge)
 
-        passwd = self.__getpassword()
+        passwd = self.__getpassword().encode('ascii')
```

Comment:
@dolohow I agree to bump to v3.5+. However, I have not idea what encoding is used in this case. I guess that while v3.5+ is fixing the type issue it does not fix the encoding issue.